src/containers/CollectionLive.js

```diff
@@ -143,57 +120,44 @@ const CollectionOnSale = ({ className, space, type, collection }) => {
       .subscribe(setOpenOrders);
 
     const fetchInscriptions = async () => {
-      try {
-        const chunkSize = 10;
-        const totalInscriptions = collection.inscriptions.length;
+      const chunkSize = 50;
 
+      const totalInscriptions = collection.inscriptions.length;
+
+      try {
+        const promises = [];
         for (let i = 0; i < totalInscriptions; i += chunkSize) {
           const inscriptionChunk = collection.inscriptions.slice(
             i,
             i + chunkSize
           );
+          promises.push(
+            getNostrInscriptions(inscriptionChunk.map((i) => i.id))
+          );
+        }
+
+        const nostrInscriptionsChunks = await Promise.all(promises);
+
+        const nostrInscriptions = nostrInscriptionsChunks.flat();
+
+        const inscriptionsWithEvents = collection.inscriptions.filter((i) =>
+          nostrInscriptions.some((ni) => ni.inscriptionId === i.id)
+        );
 
-          const inscriptionPromises = inscriptionChunk.map(
-            async (_inscription) => {
-              try {
-                const inscription = await getInscriptionData(_inscription);
-
-                try {
-                  const nostrData = await getNostrInscription(inscription);
-                  if (!nostrData) {
-                    throw new Error("Nostr data not found");
-                  }
-
-                  const { spent } = await isInscriptionSpent(inscription);
```

Comment:
i think this logic never got put back in, hence all those 10k sat listings showing up on the collection page?